pyproject.toml

```diff
@@ -75,9 +75,14 @@ count_commits_from_version_file = true
 dev_template = "{tag}.dev{ccount}"
 dirty_template = "{tag}.dev{ccount}"
 
+[tool.setuptools]
```

Comment:
Why do you need these changes?

Reply:
Needed to correctly install the package when you run pip install . and I need to be able to do this to create the documentation on read the docs

Reply:
I still find this change weird, but lets commit the pr.